csv_to_vanilla-i18n/csv_to_json.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const csv = require('csvtojson');
+const prompt = require('prompt-sync')();
```

Comment:
Is there some way by which we can avoid using dependencies? I want to keep the overhead to minimum since this script is very crucial and needs to be run by developers - without installing anything extra.

I found something which may help us, https://nodejs.org/en/knowledge/command-line/how-to-prompt-for-command-line-input/